service/message/manager_impl.go

```diff
@@ -193,6 +194,16 @@ func (m *manager) Pin(id uuid.UUID, userID uuid.UUID) (*model.Pin, error) {
 		return nil, ErrChannelArchived
 	}
 
+	// チャンネルに100個以上のメッセージがピン留めされていないか確認
+	pinLimit := 100 // ピン留めの上限数
```

Comment:
後々上限数を変更するとき一発でわかるように、変数でピン留めの上限を宣言する形にしました。constの方が良かった説はある。
```suggestion
	const pinLimit = 100 // ピン留めの上限数
```
いらないかもな～とは思っているので、もしいらなそうなら指摘お願いします。
あとはわかりやすいところに宣言だけ動かしてもいいかも？

Reply:
いや、それの方が分かりやすいので良い書き方です。
これはconstにして良いですね